ynput/ayon-core
# Provided unique file prefix in Extract OTIO review

## Changelog Description
Limits overwrites from multiple instances when creating intermediate files for burnins creation.

## Additional info
Previously it resulted in `tempFile_burnin` file which got overwritten. Now it would be containing folder path and name, eg.
`shots_mk020_mk020_0010_mk020_0010_h264.mp4`.

This file is temporary and will be correctly renamed in integration, but if multiple instances would be creating review, they would be overwriting same temporary file.

## Testing notes:
1. publish from Hiero
2. check output of `ExtractBurnin`, look for `full_input_path`, it shouldnt contain `tempFile` but folder path and name
![image](https://github.com/user-attachments/assets/06599dec-aff6-44c8-bf73-6d0a866f0900)
